build.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+#
+# Setup nginx reverse proxy via Docker.
+#
+set -eu
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/resources/builder.inc.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+################################ Main script ################################
+
+function _get_docker_image_id() {
+  echo "$(docker images -q reverse-proxy)"
+}
+
+function _get_docker_container_id() {
+  echo "$(docker ps -a -q --filter ancestor=reverse-proxy)"
+}
+
+function _stop_docker_container() {
+  KEYMAN_CONTAINER=$(_get_docker_container_id)
+  if [ ! -z "$KEYMAN_CONTAINER" ]; then
+    docker container stop $KEYMAN_CONTAINER
+  else
+    echo "No Docker container to stop"
+  fi
+}
+
+builder_describe \
+  "Setup nginx reverse proxy site to run via Docker." \
+  configure \
+  clean \
+  build \
+  start \
+  stop \
+
+builder_parse "$@"
+
+# This script runs from its own folder
+cd "$REPO_ROOT"
+
+if builder_start_action configure; then
+  # Nothing to do
+  builder_finish_action success configure
+fi
```

Comment:
```suggestion
builder_run_action configure true # nothing to do
```

`configure` should be after `clean`, structurally